src/components/App.js

```diff
@@ -4,11 +4,20 @@
 // import pokemon from '../data/pokemon/pokemon.js';
 // console.log(pokemon);
 
+import Titulo from "./Titulo.js";
+
 const App = () => {
-  const el = document.createElement('div');
+const el = document.createElement('div');
+  
+  el.className = 'tituloMemo';
+  el.appendChild(Titulo());
```

Comment:
Tal vez en vez de nombrar a la variable `el` podría tener un nombre un poco más descriptivo.
```suggestion
const containerTitle = document.createElement('div');
  
  containerTitle.className = 'tituloMemo';
  containerTitle.appendChild(Titulo());
```